### ISSUE TITLE
Have a basic website for Orbit

### ISSUE THREAD

**Issue by haadcode** [haadcode/orbit#146]
We should have a simple landing page for Orbit from which people can download client builds and find more information (links to Github, etc.)

**Comment by RichardLitt** [comment]
Agreed. We should also grab a domain for this. It can be something like dist.ipfs.io, but just for Orbit.

**Comment by theobat** [comment]
I can help for this, both regarding content and code. Humbly of course, I do not -yet- have a very deep understanding of orbit or the ipfs project in general.

**Comment by haadcode** [comment]
@theobat that would be awesome! I don't have a spec yet as to what the website should be, but gimme couple of weeks to come up with a draft that we can then iterate together. In general, I think what we need atm is just a simple landing page with screenshots and links to Github and to download the build, nothing super fancy. Sound good?

**Comment by haadcode** [comment]
After some discussions during our roadmap planning, the website is defined to contain:
  - "about page" 
  - links to code
  - links to download
  - link to web version
  - links to ipfs
  - USPs
  - screenshots + video
  - links to orbit-db
  - terms of service
  - "splash"
  - feedback button
  - list of features
  - how does it work
  - link to FAQ
  - "made by protocol labs" label
  - design for the page(s) will come from @dignifiedquire

**Comment by theobat** [comment]
@haadcode, sounds good.
Is there a tech stack defined for the website already ? (I suppose react and aegir structure from @dignifiedquire will be used but any other requirements ?). I'm down for participating but I probably need a draft or directions to make one so that I'm not doing things in the wrong way.

**Comment by haadcode** [comment]
@theobat not sure how this should be implemented tech-wise. @dignifiedquire what's the plan for the other websites we're doing? Simple static sites deployed via IPFS or something more complex?

**Comment by haadcode** [comment]
This goal depends on m writing the content for the website (https://github.com/haadcode/orbit/issues/157) which is not scheduled until end of the year. Before that, we also need to finish the product design as that will be the basis for the website content (https://github.com/haadcode/orbit/issues/149).

**Comment by RichardLitt** [comment]
Regarding the tech stack, I think that @victorbjelkholm was thinking of making a document outlining how sites should be made. I've been using Hugo.

**Comment by VictorBjelkholm** [comment]
@haadcode yeah, as richard mentioned, ipld, libp2p and multiformats are planned to continue/migrate to hugo to be fully static with all assets and deployed/served via IPFS, DNS updated with dnslink-deploy.

**Comment by milesalex** [comment]
Need design and dev help? Would love to get involved!

**Comment by RichardLitt** [comment]
@milesalex Awesome, thanks! Keep watching this - we'll be working on the website soon. @VictorBjelkholm will be the pointman it looks like.

**Comment by VictorBjelkholm** [comment]
@RichardLitt don't think anything have been decided about that, but if it's fine for @haadcode, it would make sense for me to do that.

**Comment by haadcode** [comment]
@VictorBjelkholm so if I understand you correctly, you'll be making the IPLD/multiformats/etc website with Hugo some time soon?

What would make sense here is, since we're not in a rush for Orbit website, that you work out the implementation for the other website (which I assume will produce a common way to implement the websites) and once we have one or more of them ready, the volunteers here can use those websites as a baseline for the Orbit website. meaning, @milesalex or @theobat can implement it using eg. IPLD website as a reference. I would love it if the new people in the community to have an opportunity to work on this.

Would that make sense y'all?

**Comment by VictorBjelkholm** [comment]
@haadcode yeah, that's what I'm currently working on.

Yeah, that sounds like a great idea to me :)
